hrs-portlets-api/src/main/java/edu/wisc/hr/dao/approvals/ApprovalsCountDao.java

```diff
@@ -0,0 +1,24 @@
+/**
+ * Licensed to Jasig under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work
+ * for additional information regarding copyright ownership.
+ * Jasig licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file
+ * except in compliance with the License. You may obtain a
+ * copy of the License at:
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package edu.wisc.hr.dao.approvals;
+
+public interface ApprovalsCountDao{
+    public int getApprovalsCount(String emplId);
```

Comment:
`get` methods that aren't JavaBean accessors make me sad. Maybe just `approvalsCount(String emplId)` ?

Reply:
Or something that more strongly describes the relationship of `emplId` to these approvals? `approvalsWaitingOn(String emplId)`?